src/regtech_data_validator/create_schemas.py

```diff
@@ -185,32 +185,38 @@ def get_scope_counts(schema_errors: list[SchemaError]):
     singles = [
         error for error in schema_errors if isinstance(error.check, SBLCheck) and error.check.scope == 'single-field'
     ]
-    single_errors = sum([(~error.check_output).sum() for error in singles if error.check.severity == Severity.ERROR])
-    single_warnings = sum(
-        [(~error.check_output).sum() for error in singles if error.check.severity == Severity.WARNING]
+    single_errors = int(
+        sum([(~error.check_output).sum() for error in singles if error.check.severity == Severity.ERROR])
+    )
+    single_warnings = int(
+        sum([(~error.check_output).sum() for error in singles if error.check.severity == Severity.WARNING])
     )
     multi = [
         error for error in schema_errors if isinstance(error.check, SBLCheck) and error.check.scope == 'multi-field'
     ]
-    multi_errors = sum([(~error.check_output).sum() for error in multi if error.check.severity == Severity.ERROR])
-    multi_warnings = sum([(~error.check_output).sum() for error in multi if error.check.severity == Severity.WARNING])
-    register_errors = sum(
-        [
-            (~error.check_output).sum()
-            for error in schema_errors
-            if isinstance(error.check, SBLCheck) and error.check.scope == 'register'
-        ]
+    multi_errors = int(sum([(~error.check_output).sum() for error in multi if error.check.severity == Severity.ERROR]))
+    multi_warnings = int(
+        sum([(~error.check_output).sum() for error in multi if error.check.severity == Severity.WARNING])
+    )
+    register_errors = int(
+        sum(
+            [
+                (~error.check_output).sum()
+                for error in schema_errors
+                if isinstance(error.check, SBLCheck) and error.check.scope == 'register'
+            ]
+        )
     )
 
     return Counts(
         single_field_count=single_errors,
         multi_field_count=multi_errors,
         register_count=register_errors,
-        total_count=sum([single_errors, multi_errors, register_errors]),
+        total_count=int(sum([single_errors, multi_errors, register_errors])),
```

Comment:
Don't need to cast here because single_errors, multi_errors and register_errors are already ints.  The reason the above cast has to happen is the sum([(~error.check_output).sum()...]) stuff is returning an int64 from sum(). The sum method returns the type provided here you're summing ints, so an int is returned.